ui/core/components/gear_picker.tsx

```diff
@@ -125,7 +125,7 @@ export class ItemRenderer extends Component {
 	readonly socketsContainerElem: HTMLElement;
 
 	constructor(parent: HTMLElement, player: Player<any>) {
-		super(parent, 'item-picker-root');
+		super(parent, 'item-renderer-root');
```

Comment:
The nested `item-picker-root` class was causing rendering issues particularly on mobile